xml/mysql_banners.xml

```diff
@@ -1107,8 +1208,9 @@
     <param pos="0" name="service.product" value="Percona Server"/>
     <param pos="0" name="os.family" value="Linux"/>
     <param pos="0" name="os.product" value="Linux"/>
+    <param pos="0" name="os.device" value="General"/>
   </fingerprint>
-  <fingerprint pattern="^(\d{1,2}\.\d{1,3}\.\h{1,3}) Complete MySQL by Server Logistics(?:-log)?$">
+  <fingerprint pattern="^(\d{1,2}\.\d{1,3}\.[a-fA-F\d]{1,3}) Complete MySQL by Server Logistics(?:-log)?$">
```

Comment:
Rather than using `[a-fA-F]`, it might be better to make the whole regex case insensitive.  That simplifies it slightly and heads off the almost inevitable case in the future where a few characters differ in case and this misfingerprints as a result.